Corona-Warn-App/src/main/res/values/dimens.xml

```diff
@@ -145,6 +145,6 @@
     <!-- Contact Diary -->
     <dimen name="contact_diary_list_item">64dp</dimen>
     <dimen name="contact_diary_nested_recyclerview_margin">60dp</dimen>
-    <dimen name="statistics_card_height">328dp</dimen>
+    <dimen name="statistics_card_height">352dp</dimen>
```

Comment:
IMHO this only mitigates the problem, one pixel phones you are quite limited regarding the font size. I'll test it on a Samsung device.

Reply:
still working on a better layout fix